test/flatbuffer/README.md

```diff
@@ -0,0 +1,17 @@
+#### Running flatbuffer tests
+
+Note: **Work in Progress** (Not all tests run as of now.)
+
+To run all tests in RunTests for flatbuffers, first all flatbuffer data files need to be created. 
+For this, just replacing `../../build/vowpalwabbit/vw` with `../../build/utl/flatbuffer/to_flatbuff` 
+would work, provided some flags are removed. This includes `skips`, `ngrams`, etc. A list of 
+these is present in `create_bash_script.py` which is used to create the file `create_datafiles.sh`.
```

Comment:
if `create_datafiles.sh` is a generated file, should it also be checked in?

Reply:
Yes. It has some extra commands for copying reference files (stderr, predict, etc). Although, I could generate these from the Python file as well.